Added documentation for inferred spans configuration (#1049)
elastic/apm-agent-php: docs/configuration.asciidoc

The logging level for `syslog` logging sink.
See <<configure-logging>> for details.

[float]
[[config-profiling-inferred-spans-enabled]]
==== `profiling_inferred_spans_enabled`

> NOTE: This feature is currently experimental,
> which means it is disabled by default
> and it is not guaranteed to be backwards compatible in future releases.

Reviewer: ```suggestion
preview::[]
```

We use "Technical preview" instead of "experimental". I also recommend using our built-in tech preview blocks. These expand to include all of the required legal jargon for tech preview features.

|============
| Environment variable name                           | Option name in `php.ini`
| `ELASTIC_APM_PROFILING_INFERRED_SPANS_MIN_DURATION` | `elastic_apm.profiling_inferred_spans_min_duration`
|============

[options="header"]
|============
| Default | Type
| `0ms`   | Duration
|============

The minimum duration of an inferred span.

Note that effective minimum duration is also affected by the sampling interval
so it is
max(<<config-profiling-inferred-spans-min-duration>>, <<config-profiling-inferred-spans-sampling-interval>>)

This configuration option supports the duration suffixes: `ms`, `s` and `m`.
For example: `100ms`.

[float]
[[config-profiling-inferred-spans-sampling-interval]]
==== `profiling_inferred_spans_sampling_interval`

[options="header"]
|============
| Environment variable name                                | Option name in `php.ini`
| `ELASTIC_APM_PROFILING_INFERRED_SPANS_SAMPLING_INTERVAL` | `elastic_apm.profiling_inferred_spans_sampling_interval`
|============

[options="header"]
|============
| Default | Type
| `50ms`  | Duration
|============

The frequency at which stack traces are gathered in order to construct inferred spans.

> The lower you set it, the more accurate the inferred spans durations will be.

Reviewer: ```suggestion
The lower this is set, the more accurate the inferred spans durations will be.
```
